cirq/google/engine/engine_test.py

```diff
@@ -1140,3 +1141,46 @@ def test_sampler(build):
                                                   '{apiVersion}'),
                              requestBuilder=mock.ANY)
     assert programs.create.call_args[1]['parent'] == 'projects/project-id'
+
+
+@mock.patch.object(discovery, 'build')
+def test_api_doesnt_retry_404_errors(build):
+    service = mock.Mock()
+    build.return_value = service
+    getProgram = service.projects().programs().get()
+    content = '{"error": {"message": "not found", "code": 404}}'.encode('utf-8')
+    getProgram.execute.side_effect = HttpError(mock.Mock(), content)
+    engine = cg.Engine(project_id='project-id')
+    with pytest.raises(cg.engine.engine.EngineException, match='not found'):
+        engine.get_program('foo')
+        assert getProgram.execute.call_count == 1
```

Comment:
this needs to be un-indented (also two more below)

Reply:
Fixed, thanks!